fluentd/configs.d/filter/k8s_record_transform.conf

```diff
@@ -4,7 +4,7 @@
   <record>
     hostname ${(kubernetes_host rescue nil) || File.open('/etc/docker-hostname') { |f| f.readline }.rstrip}
     message ${log}
-    version 1.0.6
+    version 1.1.4
```

Comment:
I see we are bumping these versions here.  Are they needed for the PR or is it just an opportunity to upgrade?